### DISCUSSION TITLE
Set a Custom Title and Icon

### DISCUSSION THREAD

**Discussion by EduardoNevesBL** [GeekInTheNorth/Stott.Optimizely.RobotsHandler#61] [General]
I would like to customize the title and icon of the Robots page. Is this currently viable? Are there alternatives or estimates to make this possible?

**Comment by GeekInTheNorth** [comment]
Hello @EduardoNevesBL ,

As this module is a Razor Class Library, it is possible to override the the razor files provided within your own solution.

In the web project of your own solution do the following:
- Create a new razor file on the following path: `/View/Robots/RobotsSiteList.cshtml`
- Copy the content into it from the [RobotsSiteList.cshtml](https://github.com/GeekInTheNorth/Stott.Optimizely.RobotsHandler/blob/main/src/Stott.Optimizely.RobotsHandler/Views/Robots/RobotsSiteList.cshtml) within this solution
- Make amends in your solution specific version of `/View/Robots/RobotsSiteList.cshtml`

You will however need to make sure you copy any future updates from the  package version of this file into your solution specific file.

Regards,
Mark
